.github/workflows/setup/action.yaml

```diff
@@ -44,6 +49,12 @@ runs:
       run: |
         find . -type f -executable -not -perm 755 -exec chmod 755 {} \;
         find . -type f -not -executable -not -perm 644 -exec chmod 644 {} \;
+    - id: setup-buildx-action
+      if: ${{ inputs.is_buildjet == 'true' }}
```

Comment:
why can't we detect this in the setup action?

Reply:
done